femitj/Quick-Credit
# #166108989 Response to Feedback

**What does this PR do?**
This PR shows the implementation of feedback from LFA

**Description of Task to be completed?**
Have the following done:

implemented the following feedback:

 - clients cannot post loan application until status is verified
 - Float monetary values to significant of two, NO-ROUNDUP
 - CreatedOn should have DATE in sql
 - move test inside server dir
 - app.js should have response in json() not send()
 - middlewares should act like middlewares(move actions to service directory)
 - add query validations for get all repaid and all current loans
 - add error message if token expires

**How should this be manually tested?**

 - After you clone the repo,
 - cd into it and run npm install to install node modules.
 - Make sure you have nodejs and express installed.
 - Afterwards, run npm start to initiate the app.
 - test endpoint above using postman with localhost:8080 as the host
 - test endpoint above using postman with https://quick-credit.herokuapp.com/ as the host

**What are the relevant pivotal tracker stories?**
#166108989

**Screenshots**

![Screenshot (46)](https://user-images.githubusercontent.com/40898511/58172753-546c5a00-7c91-11e9-9ff7-cf81fc33c3ea.png)

![Screenshot (48)](https://user-images.githubusercontent.com/40898511/58172931-d3fa2900-7c91-11e9-9780-ccdd1ad19bcc.png)

![Screenshot (47)](https://user-images.githubusercontent.com/40898511/58172979-fa1fc900-7c91-11e9-8c3a-49f8038fc6d3.png)
